NewConsumerInfo.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+namespace Training_Center_Task_15
+{
+    public class NewConsumerInfo
```

Comment:
better to use "contracts" for such situations

create class Consumer with desired fields 
and in case you need to create new consumer fill object with desired data.

something like

class user
{
name
password
}

class UserHelper
    {
        public User Generate()
        {
            var model = new User()
            {  
                name = Guid.NewGuid().ToString(),
                password = Guid.NewGuid().ToString()
            };
 Create(model);
return model;
}

Reply:
Done.